pkgs/top-level/all-packages.nix

```diff
@@ -23051,6 +23051,8 @@ in
 
   ns-3 = callPackage ../development/libraries/science/networking/ns3 { };
 
+  cling = callPackage ../applications/science/misc/root/cling.nix { };
```

Comment:
Is the `### SCIENCE / MISC` section of `all-packages.nix` and the `pkgs/applications/science/misc` directory the right place for cling?

And even if so, does it belong in the `root` subdirectory? https://github.com/NixOS/nixpkgs/issues/29661#issue-259735075 says something about packaging cling independently of the root framework.

Reply:
Also, AFAIK, within each section of `all-packages.nix`, we try to maintain alphabetical order, even if that means that related packages don't end up next to each other.

Reply:
I'm not sure about the right place. I guess Cling is more general-purpose than ROOT itself. What do you think of putting it in `pkgs/development/interpreters/cling/cling.nix` ?

Reply:
`pkgs/development/interpreters` sounds good to me.